assets/i18n/hi.json

```diff
@@ -86,6 +86,10 @@
 	"I18N_INTERACTIONS_MUSIC_PLAY": "चलाएँ",
 	"I18N_INTERACTIONS_MUSIC_PLAY_SEQUENCE": "लक्ष्य अनुक्रम चलाएँ",
 	"I18N_INTERACTIONS_MUSIC_SUBMIT": "जमा करें",
+	"I18N_INTERACTIONS_PENCILCODEEDITOR_CONFIRMATION_REQUIRED": "पुष्टिकरण आवश्यक",
+    "I18N_INTERACTIONS_PENCILCODEEDITOR_ARE_YOU_SURE_YOU_WANT_TO_RESET_YOUR_CODE?": "क्या आप वाकई कोड रीसेट करना चाहते हैं",
```

Comment:
Use tabs here (i.e. follow existing style of the file). This is unusual, but it's a part of the translatewiki integration.